src/components/explorer/IndividualBiosamples.js

```diff
@@ -60,28 +67,41 @@ ExperimentsClickList.propTypes = {
     handleExperimentClick: PropTypes.func,
 };
 
-const BiosampleDetail = ({ individual, biosample, handleExperimentClick }) => {
-    const resourcesTuple = useIndividualResources(individual);
+const BiosampleDetail = ({ biosample, handleExperimentClick }) => {
     return (
         <Descriptions bordered={true} column={1} size="small" style={{maxWidth: 800}}>
             <Descriptions.Item label="ID">
                 {biosample.id}
             </Descriptions.Item>
+            <Descriptions.Item label="Derived from ID">
+                {/* link to biosample */}
+                {biosample.derived_from_id ? biosample.derived_from_id : EM_DASH}
+            </Descriptions.Item>
             <Descriptions.Item label="Sampled Tissue">
-                <OntologyTerm resourcesTuple={resourcesTuple} term={biosample.sampled_tissue} />
+                <OntologyTerm term={biosample.sampled_tissue} />
+            </Descriptions.Item>
+            <Descriptions.Item label="Sample Type">
+                <OntologyTerm term={biosample.sample_type} />
             </Descriptions.Item>
             <Descriptions.Item label="Procedure">
-                <BiosampleProcedure resourcesTuple={resourcesTuple} procedure={biosample.procedure} />
+                <BiosampleProcedure procedure={biosample.procedure} />
             </Descriptions.Item>
             <Descriptions.Item label="Histological Diagnosis">
-                <OntologyTerm resourcesTuple={resourcesTuple} term={biosample.histological_diagnosis} />
+                <OntologyTerm term={biosample.histological_diagnosis} />
+            </Descriptions.Item>
+            <Descriptions.Item label="Pathological Stage">
+                <OntologyTerm term={biosample.pathological_stage} />
             </Descriptions.Item>
-            <Descriptions.Item label="Ind. Age At Collection">
-                {biosample.individual_age_at_collection
-                    ? biosample.individual_age_at_collection.age ??
-                    `Between ${biosample.individual_age_at_collection.start.age}` +
-                    `and ${biosample.individual_age_at_collection.end.age}`
-                    : EM_DASH}
+            <Descriptions.Item label="Time of Collection">
+                <TimeElement timeElement={biosample.time_of_collection}/>
+            </Descriptions.Item>
+            <Descriptions.Item label="Measurements">
+                {biosample.hasOwnProperty("measurements") &&
+                    Object.keys(biosample.measurements).length ? (
+                        <JsonView inputJson={biosample.measurements}/>
```

Comment:
this should use some kind of nice view, not a JSON view - either share components here or add a ticket for bento 16/17 to improve this + put a TODO

Reply:
Added a non routed MeasurementsTable component here